### ISSUE TITLE
Accessibity improvements needed on the Concept Page

### ISSUE THREAD

**Issue by miguelahonen** [NatLibFi/Skosmos#1038]
[Make the changes as shown in the illustration]

1) The Concept Page should offer a possibility to tab through interactive items in a logical order [numbered order in the illustration].

2) The semantic structure of the page should be improved by relevant and comprehensive heading levels [as shown in the illustration]

3) Skip link is a shortcut to important parts of the page (content) and makes the access to content quicker. The skip link helps to avoid unnecessary repetition of static page elements [content area marked in the illustration / the skip link should be located in the beginning of the body tag]
![Käsitesivu_valmis](https://user-images.githubusercontent.com/50104570/90411496-8fd41f00-e0b4-11ea-9223-05973a6fe3a5.jpg)

**Comment by miguelahonen** [comment]
- The prefferred term should be the first sentence to read for the screen reader (currently in the title)

Skip Link:
As in the picture but excluding the Finna Widget

Tab order:
- Concept description area
- Search area
- Left side list
- Finna widget
- Top menu + languages

Header levels:
?

**Comment by miguelahonen** [comment]
The skip link should become visible when it receives keyboard focus. Maybe hiding it off screen and then to be positioned on screen when it receives keyboard focus?

**Comment by miguelahonen** [comment]
All the information for a screen reader should be available in Finnish, Swedish and English

**Comment by osma** [comment]
This is relevant for Skosmos 2 only.

We will make a new accessibility review for Skosmos 3, see #1871

Closing this issue as it is no longer relevant for current development.
